config/routes.rb

```diff
@@ -19,4 +19,7 @@
   resources :service_providers
 
   root to: 'home#index'
+
+  # preserve old Groups route
+  match '/groups/:id', to: redirect('/teams/%{id}'), via: :get
```

Comment:
Sorry about that, this was me: https://github.com/18F/identity-dashboard/pull/276#discussion_r370419070

Reply:
np. this is just easier than fixing all the hubspot links.